frontend/src/toolbar/actions/ActionsEditingToolbarMenu.tsx

```diff
@@ -20,14 +20,8 @@ export const ActionsEditingToolbarMenu = (): JSX.Element => {
         editingSelectorValue,
         actionForm,
     } = useValues(actionsTabLogic)
-    const {
-        setActionFormValue,
-        selectAction,
-        inspectForElementWithIndex,
-        deleteAction,
```

Comment:
deleteAction silently failed with a 405. the API doesn't allow it and folk can jump from here to posthog if they want to delete things... but since this is broken in prod, let's just remove it